src/Operation/Aggregate.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace MongoDB\Operation;
+
+use MongoDB\FeatureDetection;
+use MongoDB\Driver\Command;
+use MongoDB\Driver\Server;
+use MongoDB\Exception\InvalidArgumentException;
+use MongoDB\Exception\InvalidArgumentTypeException;
+use MongoDB\Exception\RuntimeException;
+use ArrayIterator;
+use stdClass;
+use Traversable;
+
+/**
+ * Operation for the aggregate command.
+ *
+ * @api
+ * @see MongoDB\Collection::aggregate()
+ * @see http://docs.mongodb.org/manual/reference/command/aggregate/
+ */
+class Aggregate implements Executable
+{
+    private static $wireVersionForCursor = 2;
```

Comment:
Before you ask, this is the closest thing to a private constant. I'd rather not have this in the public API.